setup.py

```diff
@@ -46,6 +46,7 @@
     [console_scripts]
         nefertari.index = nefertari.scripts.es:main
         nefertari.post2api = nefertari.scripts.post2api:main
+        nefertari.scaffold_test = nefertari.scripts.scaffold_test:main
```

Comment:
why is it necessary to expose `scaffold_test` in console_scripts? If not, I would remove this line.